### PR TITLE
Add 7.5(.0)

### PR DESCRIPTION
**Pull request by marco-miller**
./README.md: not sure how this file should be updated exactly; maybe after PR merge? It needs potential updates I think for the TAG list and new (7.5) size in MB.

### CODE REVIEW

**Comment by marco-miller** [comment]
Running `make` test/all locally passed for me; => `"testcluster"` in green.

**Comment by blacktop** [comment]
Yeah you would add another line to the TAGs list, but don't manually update the sizes as the `make size` mcd will automatically update them.

File: 7.5/hooks/post_push [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -e
+
+VERSION=$(cat Dockerfile | grep '^ENV VERSION' | cut -d" " -f3)
+TAGS=($VERSION 7)
```
Comment (blacktop) [inline 7.5/hooks/post_push]: One last not obvious thing to do is to remove the `7` from the previous version of this file in the 7.4 folder.  This is a post build docker hook that will also tag the 7.5 image as the 7 image (replace the previous "latest" 7.4.x)

**Comment by marco-miller** [comment]
> Yeah you would add another line to the TAGs list in the README for `7.5`, but don't manually update the sizes as the `make size` cmd will automatically update them.

Thanks for these latest tips. Done. I checked and the `make size` command did its job; size was properly replaced.

**Comment by blacktop** [comment]
LGTM! Thank you!! 👍

**Pull request merged**